packages/flutter/lib/src/services/text_input.dart

```diff
@@ -2226,7 +2226,13 @@ class _PlatformTextInputControl with TextInputControl {
   Map<String, dynamic> _configurationToJson(TextInputConfiguration configuration) {
     final Map<String, dynamic> json = configuration.toJson();
     if (TextInput._instance._currentControl != _PlatformTextInputControl.instance) {
-      json['inputType'] = TextInputType.none.toJson();
+      final Map<String, dynamic> none = TextInputType.none.toJson();
```

Comment:
If this is for web only can you do that only when `kIsWeb` is true?

Reply:
yes, have added kIsWeb